frontend/server/src/DAO/Problems.php

```diff
@@ -79,10 +79,48 @@ class Problems extends \OmegaUp\DAO\Base\Problems {
         }
     }
 
+    /**
+     * @param list<string> $authors
+     */
+    final private static function addAuthorFilter(
+        array $authors,
+        string &$sql,
+        array &$args,
+        array &$clauses
+    ): void {
+        $placeholders = array_fill(0, count($authors), '?');
+        $placeholders = join(',', $placeholders);
```

Comment:
se puede evitar declarar una variable como varios tipos? la primera vez se declara como arreglo y luego como cadena.

```suggestion
        $placeholders = join(',', array_fill(0, count($authors), '?'));
```